### PR TITLE
Konyang: Point Verdant Spaceport and Waterdocks

### PR DESCRIPTION
**Pull request by DreamySkrell**
image
image
image
image

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]

File: maps/away/away_site/konyang/point_verdant/point_verdant.dm [hunk 1]
```diff
@@ -6,6 +6,7 @@
 	suffixes = list("away_site/konyang/point_verdant/point_verdant-1.dmm","away_site/konyang/point_verdant/point_verdant-2.dmm","away_site/konyang/point_verdant/point_verdant-3.dmm")
 	spawn_weight = 1
 	spawn_cost = 1
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
```
Comment (FluffyGhoster) [inline maps/away/away_site/konyang/point_verdant/point_verdant.dm:9]: This needs to be merged after the change of the spawn guaranteed PR is merged?

Comment (DreamySkrell) [reply maps/away/away_site/konyang/point_verdant/point_verdant.dm:9]: oops - good catch

Comment (DreamySkrell) [reply maps/away/away_site/konyang/point_verdant/point_verdant.dm:9]: I forgot to remove the flag, it was there just for testing before I even put up the spawn guaranteed thing PR

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**